Z/z3/build_tarballs.jl

```diff
@@ -36,9 +35,12 @@ fi
 """
 
 # Bash recipe for building across all platforms
+# Patches Z3 to work around https://github.com/Z3Prover/z3/issues/6890 (closed but not fixed!)
```

Comment:
If this comment refers to the `sed` invocation below, why not move it right next to that?